### PR TITLE
Update Clean up or delete items from the Recoverable Items folder in EXO article

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1774.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/clean-up-deleted-items.md [hunk 1]
```diff
@@ -32,38 +32,79 @@ To learn more about In-Place Hold and Litigation Hold, see [In-Place Hold and Li
...
+
+   - **Purges**: Contains hard-deleted items whose deleted item retention period has expired. Users can also hard-delete items by purging items from their Recoverable Items folder. If the mailbox is on hold, hard-deleted items are preserved. This subfolder isn't visible to end-users.
+
+   - **DiscoveryHolds**: Contains hard-deleted items that have been preserved by an eDiscovery hold or a retention policy. This subfolder isn't visible to end-users.
+
+   - **SubstrateHolds**: Contains hard-deleted items from Teams and other cloud-based apps that have been preserved by a retention policy or other type of hold. This subfolder isn't visible to end-users.
+
+3. Use the **New-ComplianceSearch** cmdlet (in Security & Compliance Center PowerShell) or use the Content search tool in the compliance center to create a content search that returns items from the target user's Recoverable Items folder. You can do this by including the FolderId in the search query for all subfolders that you want to search. For example, the following query returns all messages in the Purges and eDiscoveryHolds subfolders:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/clean-up-deleted-items.md:65]:
```suggestion
3. Use the **New-ComplianceSearch** cmdlet (in Security & Compliance Center PowerShell) or use the Content Search tool in the compliance center to create a content search that returns items from the target user's Recoverable Items folder. You can do this by including the FolderId in the search query for all subfolders that you want to search. For example, the following query returns all messages in the Purges and eDiscoveryHolds subfolders:
```

File: Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/clean-up-deleted-items.md [hunk 2]
```diff
@@ -32,38 +32,79 @@ To learn more about In-Place Hold and Litigation Hold, see [In-Place Hold and Li
...
+   ```text
+   folderid:<folder ID of Purges subfolder> OR folderid:<folder ID of DiscoveryHolds subfolder>
+   ```
 
-For detailed syntax and parameter information, see [Search-Mailbox](https://docs.microsoft.com/powershell/module/exchange/search-mailbox).
+   For more information and examples about running content searches that use the folder ID property, see [Use a folder ID or documentlink to perform a targeted collection](https://docs.microsoft.com/microsoft-365/compliance/use-content-search-for-targeted-collections#step-2-use-a-folder-id-or-documentlink-to-perform-a-targeted-collection).
 
-## Use Exchange Online PowerShell to clean up the Recoverable Items folder for mailboxes that are placed on hold or have single item recovery enabled
+   > [!NOTE]
+   > If you use the **New-ComplianceSearch** cmdlet to search the Recoverable Items folder, be sure to use **Start-ComplianceSearch** cmdlet to run the search.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/clean-up-deleted-items.md:74]:
```suggestion
   > If you use the **New-ComplianceSearch** cmdlet to search the Recoverable Items folder, be sure to use the **Start-ComplianceSearch** cmdlet to run the search.
```

File: Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/clean-up-deleted-items.md [hunk 3]
```diff
@@ -32,38 +32,79 @@ To learn more about In-Place Hold and Litigation Hold, see [In-Place Hold and Li
...
+
+This scenario is fully covered in the topic [Delete items in the Recoverable Items folder of cloud mailbox's on hold](https://docs.microsoft.com/office365/securitycompliance/delete-items-in-the-recoverable-items-folder-of-mailboxes-on-hold).
 
 ## How do you know this worked?
 
-To verify that you've successfully cleaned up or deleted items from the Recoverable Items folder of a mailbox, use [Get-MailboxFolderStatistics](https://docs.microsoft.com/powershell/module/exchange/get-mailboxfolderstatistics) cmdlet the check the size of the Recoverable Items folder.
+To verify that you've successfully deleted items from the Recoverable Items folder of a mailbox, use the [Get-MailboxFolderStatistics](https://docs.microsoft.com/powershell/module/exchange/get-mailboxfolderstatistics) cmdlet in [Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/exchange-online-powershell) to check the size and number of items in Recoverable Items folder. You can compare these statistics with the ones you collected in Step 1.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/clean-up-deleted-items.md:98]:
```suggestion
To verify that you've successfully deleted items from the Recoverable Items folder of a mailbox, use the [Get-MailboxFolderStatistics](https://docs.microsoft.com/powershell/module/exchange/get-mailboxfolderstatistics) cmdlet in [Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/exchange-online-powershell) to check the size and number of items in the Recoverable Items folder. You can compare these statistics with the ones you collected in Step 1.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**